docs/getting_started/how_to_update_mautic.rst

```diff
@@ -29,13 +30,14 @@ Mautic can only be updated using Composer via the command line from version 5.0.
 The update feature within the Mautic User interface (UI) has been deprecated from Mautic 4.2, but you will be alerted within the UI (see below figure) when a new version of the Mautic is available. 
 
 .. image:: images/gui-update-deprecated.png
-  :width: 400
+  :width: 700
+  :height: 200
   :alt: Screenshot showing deprecated update feature warning
 
 .. warning::
-    Before starting to upgrade, it is highly recommended to take a backup of your instance. If updates are available, you will receive an update notification followed by step-by-step instructions in the CLI to complete the process.
+    Before starting to upgrade, it's highly recommended to take a backup of your instance. If updates are available, you will receive an update notification followed by step-by-step instructions in the command-line interface to complete the process.
 
-Log in via the command line, and change directory to the Mautic directory using the command
+Log in via the command line, and change directory to the Mautic directory using the command:
 
 ``cd /your/mautic/directory``
```

Comment:
Should we use the proper code-block here? Or is there value in just using the inline code markup?

Reply:
inline code markup works